src/WingetCreateCLI/Commands/CacheCommand.cs

```diff
@@ -0,0 +1,99 @@
+    using Microsoft.WingetCreateCLI.Telemetry.Events;
+    using Microsoft.WingetCreateCore;
+
+    /// <summary>
+    /// Command to manage downloaded installers found in the %TEMP%/wingetcreate folder.
+    /// </summary>
+    [Verb("cache", HelpText = "CacheCommand_HelpText", ResourceType = typeof(Resources))]
+    public class CacheCommand : BaseCommand
+    {
+        /// <summary>
+        /// Gets or sets a value indicating whether to delete all downloaded installers found in cached.
+        /// </summary>
+        [Option('c', "clean", SetName = nameof(Clean), Required = true, HelpText = "Clean_HelpText", ResourceType = typeof(Resources))]
+        public bool Clean { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether to list all downloaded installers found in cache.
+        /// </summary>
+        [Option('l', "list", Required = true, SetName = nameof(List), HelpText = "List_HelpText", ResourceType = typeof(Resources))]
+        public bool List { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether to open the cache folder containing all the downloaded installers.
+        /// </summary>
+        [Option('o', "open", Required = true, SetName = nameof(Open), HelpText = "Open_HelpText", ResourceType = typeof(Resources))]
+        public bool Open { get; set; }
+
+        /// <summary>
+        /// Executes the cache command flow.
+        /// </summary>
+        /// <returns>Boolean representing success or fail of the command.</returns>
+        public override Task<bool> Execute()
+        {
+            CommandExecutedEvent commandEvent = new CommandExecutedEvent
+            {
+                Command = nameof(CacheCommand),
+            };
+
+            try
+            {
+                if (!Directory.Exists(PackageParser.InstallerDownloadPath))
+                {
+                    Directory.CreateDirectory(PackageParser.InstallerDownloadPath);
+                }
+
+                if (this.Clean)
+                {
+                    Logger.InfoLocalized(nameof(Resources.CleaningInstallers_Message), PackageParser.InstallerDownloadPath);
+                    DirectoryInfo dir = new DirectoryInfo(PackageParser.InstallerDownloadPath);
+
+                    foreach (FileInfo file in dir.GetFiles())
+                    {
+                        file.Delete();
+                    }
+
+                    commandEvent.IsSuccessful = true;
+                }
+                else if (this.List)
+                {
+                    string[] files = Directory.GetFiles(PackageParser.InstallerDownloadPath);
+                    Logger.InfoLocalized(nameof(Resources.InstallersFound_Message), files.Length, PackageParser.InstallerDownloadPath);
+                    Console.WriteLine();
+
+                    foreach (string file in files)
+                    {
+                        Logger.Info(Path.GetFileName(file));
```

Comment:
Probably a good idea to do this for Clean too

Reply:
Added additional message for clean